tests/test_e2e.py

```diff
@@ -237,7 +263,351 @@ def test_match_both_output(self):
+
+        self.generate_pe_taxsim_output(cmd)
+
+        self.assertTrue(output_file.is_file())
+        print(f"Content of {output_file}:")
+        with open(output_file, "r") as f:
+            print(f.read())
+
+    def test_generate_taxsim35_household_with_dependent_single_parent_output(self):
+        output_file = self.output_dir / self.HOUSEHOLD_WITH_DEPENDENT_SINGLE_PARENT_TAXSIM35_OUTPUT
+        self.generate_taxsim35_output(self.input_file_household_with_dependent_single_parent, output_file)
+        self.assertTrue(output_file.is_file())
+        print(f"Content of {output_file}:")
+        with open(output_file, "r") as f:
+            print(f.read())
+
+    def test_match_household_with_dependent_single_parent_output(self):
+        taxsim35_csv = pd.read_csv(self.output_dir / self.HOUSEHOLD_WITH_DEPENDENT_SINGLE_PARENT_TAXSIM35_OUTPUT)
+        pe_taxsim_csv = pd.read_csv(
+            self.output_dir / self.HOUSEHOLD_WITH_DEPENDENT_SINGLE_PARENT_PE_TAXSIM_OUTPUT
+        )
+        input_csv = pd.read_csv(self.input_file_household_with_dependent_single_parent)
+
+        print("Input CSV:")
+        print(input_csv)
+        print("\nTAXSIM35 output:")
+        print(taxsim35_csv)
+        print("\nPolicyEngine TAXSIM output:")
+        print(pe_taxsim_csv)
+
+        # Ensure both DataFrames have the same columns
+        common_columns = set(taxsim35_csv.columns) & set(pe_taxsim_csv.columns)
+        taxsim35_csv = taxsim35_csv[list(common_columns)]
+        pe_taxsim_csv = pe_taxsim_csv[list(common_columns)]
+
+        # Ensure both DataFrames have the same column names
+        taxsim35_csv.columns = taxsim35_csv.columns.str.lower()
+        pe_taxsim_csv.columns = pe_taxsim_csv.columns.str.lower()
+
+        # Sort both DataFrames by taxsimid to ensure rows are in the same order
+        taxsim35_csv = taxsim35_csv.sort_values("taxsimid").reset_index(
+            drop=True
+        )
+        pe_taxsim_csv = pe_taxsim_csv.sort_values("taxsimid").reset_index(
+            drop=True
+        )
+        input_csv = input_csv.sort_values("taxsimid").reset_index(
+            drop=True
+        )
+
+        # Convert numeric columns to float
+        numeric_columns = taxsim35_csv.select_dtypes(
+            include=["number"]
+        ).columns
+        for col in numeric_columns:
+            taxsim35_csv[col] = pd.to_numeric(
+                taxsim35_csv[col], errors="coerce"
+            )
+            pe_taxsim_csv[col] = pd.to_numeric(
+                pe_taxsim_csv[col], errors="coerce"
+            )
+
+        # Compare
+        standard_output_cols = [
+            "year",
+            # "fiitax",
+            "siitax"
+        ]
+        full_output_cols = standard_output_cols + [
+            # "tfica"
+            # "v10",  # state_agi
+            # "v13",
+            # "v18",
+            # "v19",
+            # "v26",
+            # "v28",
+            # "v34",
+            # "v45",
```

Comment:
why commented?

Reply:
those kept to re-enable again after discussing with tax for the use_tax usage which seems taxsim side does not use